docs/guides/telemetryservice/pushtoMinIO.md

```diff
@@ -0,0 +1,70 @@
+# To MinIO
+
+## Create TelemetryService Yaml file
+```yaml
+#telemetry_service.yaml
+apiVersion: shifu.edgenesis.io/v1alpha1
+kind: TelemetryService
+metadata:
+  name: push-file-mp4
+  namespace: devices
+spec:
+  telemetrySeriveEndpoint: http://telemetryservice.shifu-service.svc.cluster.local
+  serviceSettings:
+    # the timeout(ms) when deviceShifu send to TelemetryService
+    RequestTimeout: 20000
+    MinIOSetting:
+      # the bucket which you want to upload to
+      Bucket: test-bucket
+      # your file's extension
+      FileExtension: mp4
+      # minio service's address
+      EndPoint: minio.data.svc.cluster.local:9000
+      # you can specify a secret or APIId and APIKey
+      Secret: minio-secret
+```
+
+## Create Secret
+
+you need fill the `username` and `password` field, if you don't have a AccessId and AccessKey,
```

Comment:
"you can use either MinIO's username and password or the AccessId and AccessKey created in the MinIO GUI"